src/codemagic/apple/resources/resource.py

```diff
@@ -202,7 +202,10 @@ def from_iso_8601(cls, iso_8601_timestamp: str) -> datetime:
     def from_iso_8601(cls, iso_8601_timestamp: Optional[str]):
         if iso_8601_timestamp is None:
             return None
-        return datetime.strptime(iso_8601_timestamp, '%Y-%m-%dT%H:%M:%S.%f%z')
+        try:
+            return datetime.strptime(iso_8601_timestamp, '%Y-%m-%dT%H:%M:%S.%f%z')
+        except ValueError:
+            return datetime.strptime(iso_8601_timestamp, '%Y-%m-%dT%H:%M:%S%z')
```

Comment:
iso 8601 without milliseconds in /builds api

```
            ...
            "uploadedDate":"2021-01-28T06:01:32-08:00",
            "expirationDate":"2021-04-28T06:01:32-07:00",
            ...
```

Reply:
added [here](https://github.com/codemagic-ci-cd/cli-tools/pull/67/files#diff-b736b487114b15d1320d4be383e9cb605e79ac0e683d6949d012fc6215dcf44eR204)

Reply:
The comment would be better suited in `except` block.